starknet_py/net/signer/test_stark_curve_signer.py

```diff
@@ -57,3 +57,29 @@ def test_sign_transaction(transaction):
     assert len(signature) > 0
     assert all(isinstance(i, int) for i in signature)
     assert all(i != 0 for i in signature)
+
+
+def test_keypair():
+    key_pair = KeyPair(private_key="0x123", public_key="0x456")
+    assert isinstance(key_pair.public_key, int)
+    assert isinstance(key_pair.private_key, int)
+
+    key_pair = KeyPair(private_key="0x123", public_key=0x456)
+    assert isinstance(key_pair.public_key, int)
+    assert isinstance(key_pair.private_key, int)
+
+    key_pair = KeyPair(private_key=0x123, public_key="0x456")
+    assert isinstance(key_pair.public_key, int)
+    assert isinstance(key_pair.private_key, int)
+
+    key_pair = KeyPair(private_key=0x123, public_key=0x456)
+    assert isinstance(key_pair.public_key, int)
+    assert isinstance(key_pair.private_key, int)
+
+    key_pair = KeyPair.from_private_key("0x123")
+    assert isinstance(key_pair.public_key, int)
+    assert isinstance(key_pair.private_key, int)
+
+    key_pair = KeyPair.from_private_key(0x123)
+    assert isinstance(key_pair.public_key, int)
+    assert isinstance(key_pair.private_key, int)
```

Comment:
I don't think it's that necessary of a test 😄

Reply:
It was supposed to be more of a codecov targeted test, but he didn't like it
![image](https://github.com/software-mansion/starknet.py/assets/23462504/58d88ab9-7fc0-4610-8b8a-2bb1dc8d17a6)
Will delete it then